src/common/metrics/vaccinations.tsx

```diff
@@ -122,7 +121,10 @@ function renderStatus(projections: Projections): React.ReactElement {
       {bivalentFall2022BoosterText}. Anybody who is at least 6 months old is
       eligible to be vaccinated. Fewer than 0.001% of people who have received a
       dose experienced a severe adverse reaction. {cappedVaccinatedCopy}
-      <Link to="/faq#vaccines">See more vaccine resources and FAQs</Link>.
+      <a href="https://www.cdc.gov/coronavirus/2019-ncov/vaccines/about-vaccines/index.html">
```

Comment:
I don't know how much it matters, but we have an `ExternalLink` component I think we usually use...

Reply:
ah ok--will update! 🙏